docs/priority/README.md

```diff
@@ -1,18 +1,26 @@
 # Workload Priority
 In Kubernetes, assigning different priorities to workloads ensures efficient resource management, minimizes service disruption, and supports better scaling. 
-It allows critical applications to receive the resources they need before less important ones, ensuring high availability and compliance with SLAs. 
-By prioritizing workloads, KAI Scheduler can preempt lower-priority workloads to make room for higher-priority ones. 
+It allows critical applications to receive the resources they need before less important ones, ensuring high availability and compliance with SLAs.
+By prioritizing workloads, KAI Scheduler schedules jobs according to their assigned priority. 
+When sufficient resources aren't available for a workload, the scheduler can preempt lower-priority workloads to free up resources for higher-priority ones.
 This approach ensures that mission-critical services are always prioritized in resource allocation.
 
+## Priority Classes
+KAI scheduler deployment comes with several predefined priority classes:
+1. `train` (50) - can be used for preemptible training workloads
+2. `build-preemptible` (75) - can be used for preemptible build/interactive workloads
+3. `build` (100) - can be used for build/interactive workloads (non-preemptible)
+4. `inference` (125) - can be used for inference workloads (non-preemptible) 
```

Comment:
Maybe explain in a sentence that the pod-grouper assumes default priorities for different workload types, but it can be overridden with the label?

Reply:
I will move the ##API section closer to here and will explain there how to set a custom priority